src/index.ts

```diff
@@ -0,0 +1,80 @@
+import './styles/style.css';
+import { createTemplate, IPersonData } from './template';
+import { createSession } from './xhr';
+
+interface IOptions {
+  host?: string;
+  apiKey: string;
+  parentId: string;
+  onSession: (err, response) => void;
+}
+
+interface IMountOptions {
+  formLabel?: any;
+  submitBtnText?: string;
+  loadingText?: string;
+}
+
+interface IParams {
+  person?: IPersonData;
+  vendorData?: string;
+}
+
+const Veriff = (options: IOptions) => {
+  const { host = 'https://api.veriff.me', apiKey, parentId, onSession } = options;
+  return {
+    params: {
+      person: {},
+    },
+    setParams(newParams: IParams) {
+      this.params = { ...this.params, ...newParams };
+    },
+    updateParams(newParams: IParams) {
+      this.params = { ...newParams };
```

Comment:
setParams "patches" the params with new values, but updateParams replaces them. is this intentional?